src/tests/test_graph_persistence.py

```
@@ -0,0 +1,115 @@
+from typing import Any
+from unittest.mock import MagicMock, call
```

Flake8 (7.3.0)

[error] 'unittest.mock.call' imported but unused

(F401)